### PR TITLE
Temperature and repetier FW updates

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: working.diff [hunk 1]
```diff
@@ -0,0 +1,87 @@
+diff --git a/src/ifirmware.h b/src/ifirmware.h
```
Comment (sithlord48) [inline working.diff]: What is this diff file?

File: testclient/mainwindow.ui [hunk 2]
```diff
@@ -137,6 +137,13 @@
              </widget>
             </item>
             <item>
+             <widget class="QLabel" name="bedTempLB">
```
Comment (sithlord48) [inline testclient/mainwindow.ui]: These Gui changes are not connected to anything can we do this on a commit that also contains logic to make them function.

File: src/plugins/repetierplugin.cpp [hunk 3]
```diff
@@ -1,19 +1,52 @@
...
+Q_LOGGING_CATEGORY(REPETIER_PLUGIN, "org.kde.atelier.core.firmware.repetier");
+
 QString RepetierPlugin::name() const
 {
     return QStringLiteral("Repetier");
 }
 
 RepetierPlugin::RepetierPlugin(QObject *parent)
 {
+    qCDebug(REPETIER_PLUGIN) << "Starting.. " << name();
```
Comment (sithlord48) [inline src/plugins/repetierplugin.cpp]: this just sounds weird coming from within the plugin like this maybe change the message to 
qCDebug(REPETIER_PLUGIN) << name() << " plugin loaded" 
 or something similar to that.

**Review by sithlord48 — CHANGES_REQUESTED** [review]

File: working.diff (hunk 1 above)
Comment (patrickelectric) [reply working.diff]: An error

File: src/plugins/repetierplugin.cpp (hunk 3 above)
Comment (patrickelectric) [reply src/plugins/repetierplugin.cpp]: Done

**Review by sithlord48 — APPROVED** [review]

**Pull request merged**